asreview/logging/utils.py

```diff
@@ -3,18 +3,18 @@
 
 def get_logger_class(fp):
     "Get logging class from file extension."
-    from asreview.hdf5_logging import HDF5_Logger
-    from asreview.json_logging import JSON_Logger
-    from asreview.json_logging import Dict_Logger
+    from asreview.logging.hdf5 import HDF5Logger
```

Comment:
Please move them out of the function (better performance).

Reply:
I can't; I get import errors when I do that.